community-handbook/chaosscast.md

```diff
@@ -170,22 +177,21 @@ Georg Link has an IFTT recipe to:
 
 ## Procedures
 
-These procedures are for the organizer of each episode:
+These procedures are for the organizer and coordinator of each episode:
 
 ### Scheduling an episode 
 * always
   * Think of who would make a good guest for the podcast and ask them tentatively if you may invite them
-* **Step 1:** Determine a guest and panlists
-  * **Action: **Recruit interview guest and decide topic together
-    * Ask for 60-90 minute timeslots that work for guest.
-    * Create a Doodle poll for panelists to schedule time
-  * **Action:** Prepare the planning document (copy the template and start filling in information)
-  * **Action:** send out an [invitation email](https://docs.google.com/document/d/1IniJc9o1OCBBTYv50W3WTfZYmshi4aYusHIK4ADulk8/edit) with a [Doodle](doodle.com) for finding a time
-    * Invite guest
-    * Invite panelists ([list of email addresses](https://docs.google.com/document/d/1ca9TJY17cj-cBWmJP5qjCX56133unsaOlEUwACc4PMA/edit))
+* **Step 1:** Determine a guest and panelists (Organizer)
+  * **Action:** Recruit interview guest and decide topic and content of the podcast together
+* **Step 2:** Coordinate Episode (Community Manager)
+    * **Action:** Work with guest to find a suitable time 60-90 minutes.
+    * **Action:** Find time that works for guest and organizer.
+    * **Action:** Prepare the planning document (copy the [Episode Planning Document template](https://docs.google.com/document/d/17qjAafr0aAWXV_HR-WiefHOa881t8X7tEoiZnPBh3ao/edit#heading=h.cmpmu92ylw4d) and start filling in information.)
+   * **Action:** send out an [invitation email](https://docs.google.com/document/d/1IniJc9o1OCBBTYv50W3WTfZYmshi4aYusHIK4ADulk8/edit) with a [Doodle](doodle.com) for finding a time
+   * **Action:** Invite panelists ([list of email addresses](https://docs.google.com/document/d/1ca9TJY17cj-cBWmJP5qjCX56133unsaOlEUwACc4PMA/edit))
```

Comment:
We may have to work on this step. After finding a time or time options with guest and organizer, I think the doodle poll only needs to go out to panelists. 

What do you think?

```suggestion

```

```suggestion
   * **Action:** ask panelists ([list of email addresses](https://docs.google.com/document/d/1ca9TJY17cj-cBWmJP5qjCX56133unsaOlEUwACc4PMA/edit)) for their availability with an [invitation email](https://docs.google.com/document/d/1IniJc9o1OCBBTYv50W3WTfZYmshi4aYusHIK4ADulk8/edit) and a [Doodle poll](doodle.com)
   * **Action:** Organizer and coordinator select 1-2 panelists
```

Reply:
Since it appears that the guest invitation is initially done informally by the organizer, how about this? 

 - under the Action item of "Find a time that works for the guest and organizer" perhaps this is where a revised version of the invitation email would come in, along with a request for a few potential time slots from the guest. 
- then I could check with the organizer to pick one of those time slots.
- then assuming panelists are on the podcast email, we can send an email to the podcast mailing list with the confirmed date and time and ask for availability.

Do we need a doodle at all?

Reply:
Good idea. Let's try it. 

One modification: The podcast@ email is only for organizers and coordinators. We would have to send invites to panelists separately.

**No doodle** needed since guest and organizer already agreed on a specific day and time.
A simple "are we meeting yet" to help with time zone conversion would be enough.
This would reduce the burden on the panelists to only having to reply "yes" or "no" to the invitation email.

Reply:
>One modification: The podcast@ email is only for organizers and coordinators. We would have to send invites to panelists separately.

👍 Ok got it! 

>No doodle needed

👍 

>This would reduce the burden on the panelists to only having to reply "yes" or "no" to the invitation email.

👍